### PR TITLE
Update README to add high level goals.

### PR DESCRIPTION
**Pull request by jasmussen**
This PR seeks to freshen up the README by trimming many sections, de-emphasizing prototypes, and adding a couple of new sections detailing high level goals.

Should we include an image, like https://wpcoredesign.mystagingwebsite.com/gutenberg/Text,%20Selected.png for example?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,47 +3,57 @@
...
+## Why
 
-Be sure to have Node installed first — simply follow <a href="https://nodejs.org/en/">the instructions on the Node JS website</a>. Then, in a terminal, type `npm install` to get the dependencies all set up. Once that finishes, you can type `npm run build` to make the plugin functional, or `npm run dev` to keep the plugin building in the background as you work on it. Then pick among the <a href="https://github.com/WordPress/gutenberg/issues">tickets</a> and enjoy hacking away at making the world better.
+One thing sets WordPress apart from other systems, is that it allows you to create as rich a post layout as you can imagine. But only if you know HTML & CSS and building your own custom theme. By thinking of the editor as a tool to let you write rich posts, and in a few clicks create gorgeous playful layouts, hopefully we can make people start to _love_ WordPress, as opposed to pick it because it's what everyone else uses to blog.
 
-## How Designers Can Contribute
+In that vein, `post_content` is your document, and the visual editor is Apple Pages. And you write and create beautiful posts, even if there are some blocks that don't get selected when you press ⌘A.
```
Comment (mtias) [inline README.md]: This seems a bit disconnected form the previous paragraph.

**Comment by mtias** [comment]
An image could be good, yes.

File: README.md [hunk 2]
```diff
@@ -3,47 +3,57 @@
...
 
-- <a href="https://github.com/Automattic/wp-post-grammar">WP Post grammar parser</a>.
+## How Designers Can Contribute
 
-## How Developers Can Contribute
+If you'd like to contribute to the design or front-end, feel free to contribute to tickets labelled <a href="https://github.com/WordPress/gutenberg/issues?q=is%3Aissue+is%3Aopen+label%3ADesign">Design</a>. We could useyour thoughtful replies, mockups, animatics, sketches, doodles. Proposed changes are best done as minimal and specific iterations on the work that precedes it so we can compare. If you use <a href="https://www.sketchapp.com/">Sketch</a>, you can grab <a href="https://cloudup.com/cMPXM8Va2cy">the source file for the mockups</a> (updated April 6th).
```
Comment (aduth) [inline README.md]: Space between "use" and "your".

**Comment by jasmussen** [comment]
Solid feedback all around. Tweaked the text further, fixed typo, added image. 

Good to merge?

**Pull request merged**